src/node/rpc/member_frontend.h

```diff
@@ -499,39 +606,59 @@ namespace ccf
           }
           catch (const std::logic_error& e)
           {
-            return make_error(
+            ctx.rpc_ctx->set_error(
               HTTP_STATUS_INTERNAL_SERVER_ERROR,
               ccf::errors::InternalError,
               fmt::format("Error issuing new recovery shares: {}", e.what()));
+            return;
           }
         }
-        return make_success();
+        ctx.rpc_ctx->set_response_status(HTTP_STATUS_NO_CONTENT);
+        return;
       };
-      make_endpoint("/ack", HTTP_POST, json_adapter(ack), member_sig_only)
+      make_endpoint("/ack", HTTP_POST, ack, member_sig_only)
```

Comment:
It's not clear to me why the `json_adapter` wrapper has been removed here as the payload is still JSON?

Reply:
Because the payload isn't JSON, the payload is now `application/cose`, with JSON contents.